### ISSUE TITLE
Article: Getting started in readme

### ISSUE THREAD

**Issue by prokopsimek** [DXHeroes/knowledge-base#115]

**Comment by ridlees** [comment]
What is this about: Update to the https://github.com/DXHeroes/knowledge-base-content/blob/master/practices/readme.md on how to add **Getting started** part. Examples of good code samples, installation guides, and requirements. 
Read more here: https://rowanmanning.com/posts/writing-a-friendly-readme/
-  Follow our [Contribution guide](https://github.com/DXHeroes/knowledge-base-content/blob/master/CONTRIBUTING.md)
- How to [Git and Pull Requests]( https://github.com/firstcontributions/first-contributions/blob/master/README.md)
- How to [Markdown](https://guides.github.com/features/mastering-markdown/), [Markdown interactive demo](https://www.markdowntutorial.com/lesson/1/) or [Google Docs Addon](https://gsuite.google.com/marketplace/app/docs_to_markdown/700168918607)

**Comment by ramesht007** [comment]
hey, @prokopsimek  @ridlees i would like to work on this one.

**Comment by DominikMilaniak** [comment]
Hi @ramesht007. Another two issues you want to contribute to? Awesome! Thank you so much. I will gladly assign you to this and "Multiple environments" issue - if it would turn out this is too much for you, it's ok, just contact me and I will assign the issue to anyone else 😄 We are so happy you chose to make our knowledge base better and share your knowledge with hundreds of developers and team leaders around the world, you are amazing! Looking forward to your articles! 😄

**Comment by ramesht007** [comment]
Thanks @DominikMilaniak 🙂

**Comment by ramesht007** [comment]
I am just learning stuff, and according to me blogging out what you have understood is the best one gain knowledge and share it everyone. 😁😀

**Comment by DominikMilaniak** [comment]
Yes! I couldn't agree more 😄 I am glad you chose our Knowledge Base ("blog") from all the other possibilities! 😄

**Comment by DominikMilaniak** [comment]
**Hi again** @ramesht007! 🙂 

A few days have passed since our last contact, how is it going so far with this article? I see you are working on the [Git](https://github.com/DXHeroes/knowledge-base-content/issues/184) article and you are doing absolutely amazing work there, but it looks like it took a huge portion of your schedule, so I just wanted to ask, if you still want to do this and [Multiple Environments](https://github.com/DXHeroes/knowledge-base-content/issues/166) article? Is anything not clear enough or do you need help with something? I am in no way pushing you, you can take as much time as you need 🙂  Just wanted to get in touch to make sure everything is ok and if the load is too much on you, I can assign these articles to someone else 👍

**Comment by ramesht007** [comment]
Hi @DominikMilaniak , sorry for delay i will probably make a PR tomorrow. It took me time as i was thinking about how to organise the content, so that it looks good and is readable.😀
